packages/core/admin/admin/src/translations/en.json

```diff
@@ -481,26 +481,26 @@
   "components.FilterOptions.FILTER_TYPES.$notNull": "is not null",
   "components.FilterOptions.FILTER_TYPES.$null": "is null",
   "components.FilterOptions.FILTER_TYPES.$startsWith": "starts with",
-  "components.Input.error.attribute.key.taken": "This value already exists",
+  "components.Input.error.attribute.key.taken": "The value already exists",
   "components.Input.error.attribute.sameKeyAndName": "Can't be equal",
   "components.Input.error.attribute.taken": "This field name already exists",
   "components.Input.error.contain.lowercase": "Password must contain at least one lowercase character",
   "components.Input.error.contain.number": "Password must contain at least one number",
   "components.Input.error.contain.uppercase": "Password must contain at least one uppercase character",
-  "components.Input.error.contentTypeName.taken": "This name already exists",
+  "components.Input.error.contentTypeName.taken": "The name already exists",
```

Comment:
I honestly would remove all the "the" and have for instance in this specific case "Name already exists", what do you think?

Reply:
second

Reply:
Updated in 304901ed7ef7d061fe7860ef36db60aa78ac66b2